api/servers_handler.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/topfreegames/pitaya"
+)
+
+// ServersHandler handler
+type ServersHandler struct {
+	App *App
+}
+
+// NewServersHandler creates a new servers handler
+func NewServersHandler(a *App) *ServersHandler {
+	m := &ServersHandler{App: a}
+	return m
+}
+
+// ServeHTTP method
+func (s *ServersHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	param := strings.Replace(r.URL.Path, "/servers/", "", -1)
```

Comment:
😱not the correct way of retrieving a variable with gorilla mux, check the docs (https://godoc.org/github.com/gorilla/mux)

```
r.HandleFunc("/products/{key}", ProductHandler)

vars := mux.Vars(request)
category := vars["key"]
```